protos/status/status.proto

```diff
@@ -22,12 +22,14 @@ import "google/protobuf/timestamp.proto";
 
 service Status {
   rpc UpdateMetrics(SignedMetrics) returns (google.protobuf.Empty);
+  rpc UpdateMetricsAggregation(NodeMetricsAggrReq) returns (google.protobuf.Empty);
   rpc CollectHealth(NodeHealth) returns (google.protobuf.Empty);
   rpc CollectError(NodeError) returns (google.protobuf.Empty);
 }
```

Comment:
Plz help to clean up this one, this is legacy.

Reply:
Removed, thanks